docs/.vitepress/theme/vars.css

```diff
@@ -125,10 +141,8 @@
 
 :root {
 	--vp-snippet-toggler-border: var(--vp-c-gray-light-4);
```

Comment:
Should we get rid of this var too and configure it inside the component?

Reply:
I have no strong opinions. If you do, we can.